README.Rmd

```diff
@@ -1,7 +1,5 @@
 ---
-output:
-  github_document:
-    html_preview: false
+output: github_document
```

Comment:
Why this change?

Reply:
just because I like seeing the HTML version when I render it locally, it doesn't change anything for CI or else I think